docker type in the gap (#2590)
enkidevs/curriculum: docker/docker-fundamentals/docker-images/docker-image-registries.md

docker push enki
```

> You're not required to host your images on Docker Hub and can setup your own registry as well.

Author: ```suggestion
You're not required to host your images on Docker Hub and can setup your own registry if you like.
```
